core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -31,8 +31,10 @@ interface CkeditorCustomScope extends ng.IScope {
 }
 
 angular.module('oppia').directive('ckEditor4Rte', [
-  'ContextService', 'RteHelperService', 'ENABLE_SVG_EDITOR_RTE',
-  function(ContextService, RteHelperService, ENABLE_SVG_EDITOR_RTE) {
+  'CkEditorCopyContentService', 'ContextService', 'RteHelperService',
```

Comment:
Do we need to import this service on top?

Reply:
Done